src/GraphQL/transformers/className.js

```diff
@@ -0,0 +1,39 @@
+import logger from '../../logger';
+
+const parseMap = {
```

Comment:
We will still have a problem if I create a class called FileWhereInput for example. I know it is weird, but maybe it would be better to create a method in ParseGraphQLSchema prototype called push type that would check if the type was pushed before, it will add the type name to a list and push type to the this.graphQLTypes. Then we could replace all parseGraphQLSchema.graphQLTypes.push() to a call to this new function. If you want, I can help you on this.

Reply:
I think it can't collide with `FileWhereInput` it is an input, in normal implementation `GraphQL` can make the difference between an `input` and a `type`

Reply:
It was actually a bad example, but a class called `FileInfo` would be a problem. Or a class called `Query` for example. There is also the problem we talked before of creating the class `Car` and the class `Cars`. We'd need to create a similar logic that I proposed above also for fields that are pushed to the queries and mutations types.